app/api/routers/health_check_router.py

```diff
@@ -1,17 +1,35 @@
-import logging
-from fastapi import APIRouter
-from api.models.health_check_status import HealthCheckStatus
+from typing import Callable
+from fastapi import APIRouter, Request, Response
+from fastapi.routing import APIRoute
+
+from app.api.models.health_check_status import HealthCheckStatus
+
+from app.logger_config import logger
+
+
+class LogRoute(APIRoute):
+    def get_route_handler(self) -> Callable:
+        original_route_handler = super().get_route_handler()
+
+        async def custom_route_handler(request: Request) -> Response:
+            req = request.method, request.url

```

Comment:
also since we are assigning two comma separated values like to a var - python is duck typing it to a Tuple and due to this the log line is having extra brackets. 
Instead it is enough to directly write - logger.info(f"Received {request.method} {just_the_uri} )

and here be just the uri and not the one with ip, etc.

Reply:
changes incorporated